tests/tests/tests.rs

```diff
@@ -51,8 +50,27 @@ fn overloaded_contract_functions() {
             "test",
             "--standalone-node",
             graph_node.to_str().unwrap(),
+            "--node-logs",
             "yarn test",
         ],
         dir,
     );
 }
+
+lazy_static! {
+    // Necessary due to port conflict when running multiple nodes at once.
+    // TODO: Allow tests to run in parallel.
```

Comment:
Shouldn't be that hard to do if e.g. we give `graph test` a `--start-port` and enumerate all ports in the compose file based on that. We can look into that.

Reply:
That's a nice idea.